.changeset/stale-radios-punch.md

```diff
@@ -0,0 +1,8 @@
+---
+'@commercetools-uikit/spacings-inset-squish': minor
+'@commercetools-uikit/spacings-inset': minor
+---
+
+Refactor vertical space usage so consumers can control it.
+
+There's a new property named `useAllAvailableHeight` that will force the component main container to use `100%` of the available height.
```

Comment:
👋🏼 this should change 🤗

Reply:
Thanks 👍 

Updated here: fd1624cf18d6da2cce7ddc72eedee273311450f7